vercel/tsconfig.json

```diff
@@ -0,0 +1,21 @@
+{
+  "extends": "@tsconfig/svelte/tsconfig.json",
```

Comment:
I had to remove this line for it to work locally, otherwise I would get this error when calling api/languages:

` TS5095: Option 'preserveValueImports' can only be used when 'module' is set to 'es2015' or later.`

Reply:
:thinking: That's weird, it's part of the Vercel default, so I'd assume it would be compatible with Vercel serverless functions in Typescript.
:shrug: Well, as long as its deployment is successful 😆